integration-tests/tests/run-workflow-on-snapshot.js

```diff
@@ -66,6 +66,9 @@ const testRunWorkflowOnSnapshotFn = _.flow(
   await launchWorkflowAndWaitForSuccess(page)
 
   await clickNavChildAndLoad(page, 'data')
+  // Before click "Workspace Data" link, we need to make sure page has rendered all UI elements.
+  // "testsnapshot" link can take few seconds more to render on page.
+  await findElement(page, clickable({ textContains: 'testsnapshot' }))
```

Comment:
It doesn't seem immediately obvious why we need `testsnapshot` to appear before clicking the "Workspace Data" button. Maybe worth a comment explaining that this is so that the "Workspace Data" button doesn't move when trying to click on it?